docs/src/main/sphinx/release/release-353.md

```diff
@@ -37,6 +43,7 @@
 
 * Fix `Classloader` related issue seen in Worker for Schema Registry tables. ({issue}`6902`)
 * Fix querying of Schema Registry tables with References in their schema. ({issue}`6907`)
+* Fix listing of schema registry tables having ambiguous subject name in lower case.
```

Comment:
@Praveen2112 should this be linked to https://github.com/trinodb/trino/pull/7048 ?

the PR title suggests improvement in err message "only"